docs/Ecosystem.md

```diff
@@ -18,6 +18,7 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-cookie`](https://github.com/fastify/fastify-cookie) Parse and set cookie headers.
 - [`fastify-cors`](https://github.com/fastify/fastify-cors) Enables the use of CORS in a Fastify application.
 - [`fastify-env`](https://github.com/fastify/fastify-env) Load and check configuration.
+- [`fastify-etag`](https://github.com/fastify/fastify-etag) Automatically generate etags for HTTP responses
 - [`fastify-elasticsearch`](https://github.com/fastify/fastify-elasticsearch) Plugin to share the same ES client.
```

Comment:
Can you fix the order here? It should be `fastify-elasticsearch` => `fastify-env` => `fastify-etag`

Reply:
Wow, third time's the charm :facepalm: Fixed.